packages/shared_preferences/shared_preferences/lib/src/devtools_extension.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:developer' as developer;
+
+import 'package:flutter/foundation.dart';
+
+import '../shared_preferences.dart';
+
+const String _eventPrefix = 'shared_preferences:';
```

Comment:
nit: should this be `shared_preferences.` instead of `shared_preferences:`? Using dot is a more common convention for event names than using a colon.

Reply:
I think I copied this from somewhere, gonna change it to `.`.